src/modules/otp/otp.service.ts

```diff
@@ -30,7 +31,7 @@ export class OtpService {
 
       return otp;
     } catch (error) {
-      Logger.error('OtpServiceError ~ createOtpError ~', error);
+      console.log('OtpServiceError ~ createOtpError ~', error);
```

Comment:
the response interceptor is a good point to log these errors, we can do without the try/catch

Reply:
I created a fix for it in an upcoming pr, Logging it in the interceptor actually does not specify where the error is coming from